### ISSUE TITLE
about dataset setting

### ISSUE THREAD

**Issue by cnh-boom** [FGTI2024/FGTI24#1]
```
train_loader = DataLoader(dataset, batch_size=configs.batch, num_workers=0, shuffle=True)
    
test_loader = DataLoader(dataset, batch_size=configs.batch, num_workers=0, shuffle=False)
    
return train_loader, test_loader
```

It seems that training and testing use the same data

**Comment by Reast1nPeace** [comment]
Thank you for your question.
Unlike downstream tasks such as forecasting and classification, the goal of the imputation task is to fill in the missing values in the original dataset and return a complete dataset. 
Therefore the training and test data are the same, the difference is that for training we use a self-supervised strategy (similar to mask language modeling) to mask the already missing data again (on p. 7 of the paper)
